zulip/zulip/__init__.py

```diff
@@ -331,11 +337,11 @@ def __init__(self, email=None, api_key=None, config_file=None,
                 elif insecure_setting == "false":
                     insecure = False
                 else:
-                    raise RuntimeError("insecure is set to '%s', it must be 'true' or 'false' if it is used in %s"
-                                       % (insecure_setting, config_file))
+                    raise ZulipError("insecure is set to '%s', it must be 'true' or 'false' if it is used in %s"
+                                     % (insecure_setting, config_file))
         elif None in (api_key, email):
-            raise RuntimeError("api_key or email not specified and file %s does not exist"
-                               % (config_file,))
+            raise FileNotFoundError("api_key or email not specified and file %s does not exist"
```

Comment:
We should call this `ConfigNotFoundError`, since we're not necessarily looking for a file here.

Reply:
So should we change all FileNotFoundError with ConfigNotFoundError or just this one?